### ISSUE TITLE
A11y Fail: Data Grid - Pagination fix

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1383]
CENG_1956

Data Grid: https://react-magma.cengage.com/version/4.5.0/api/datagrid/

### Issue
When operating the pagination controls, content updates to indicate the number of rows displaying (i.e. 11-20 of 25). This content can be considered a live region and should be announced to screen readers when it updates.

### Suggestion
Please add aria-live="polite" to this content so that it is announced when it updates. This will give screen reader users an indication that the content in the table updated based on the action that they took.

**Comment by chris-cedrone-cengage** [comment]
Looks good!
